src/mongocxx/options/aggregate.hpp

```diff
@@ -21,6 +21,7 @@
 #include <bsoncxx/document/view.hpp>
 #include <bsoncxx/stdx/optional.hpp>
 #include <mongocxx/read_preference.hpp>
```

Comment:
In the stdx PR, you've changed most things to have a space between the bsoncxx includes and the mongocxx includes.  Either style is fine with me, but let's be consistent.